felayout_t3kit/dev/styles/main/contentElements/heroImage.less

```diff
@@ -41,20 +41,24 @@
 }
 
 .hero-image__caption {
-    padding: 20px 60px;
+    padding: 15px;
     color: @main-body-bg;
     overflow: hidden;
     -webkit-font-smoothing: antialiased;
 }
 
 .hero-image__caption .hero-image__caption-header {
     color: @main-body-bg;
-    font-style: normal;
+    font-style: italic;
+    margin-top: 5px;
+    font-size: @font-size-h2;
 }
 
 .hero-image__caption .hero-image__caption-subheader {
```

Comment:
Could you explain why we need both classes? I would suggest using only one `.hero-image__caption-subheader`